SQL/0000-00-05-ElectrophysiologyTables.sql

```diff
@@ -0,0 +1,289 @@
+  VALUES
+  ('raw',         'raw dataset'),
+  ('derivatives', 'derivative/processed dataset');
+
+-- Insert into physiological_channel_type
+INSERT INTO physiological_channel_type
+  (`ChannelTypeName`, `ChannelDescription`)
+  VALUES
+  ('EEG',              'ElectoEncephaloGram: EEG sensors'                    ),
+  ('VEOG',             'Vertical ElectroOculoGram (eyes)'                    ),
+  ('HEOG',             'Horizontal ElectOculoGram'                           ),
+  ('EOG',              'Generic EOG channel, if HEOG or VEOG information not available'),
+  ('ECG',              'ElectroCardioGram (heart)'                           ),
+  ('EMG',              'ElectroMyoGram (muscle)'                             ),
+  ('TRIG',             'System Triggers'                                     ),
+  ('REF',              'Reference electrode'                                 ),
+  ('GRD',              'Ground electrode'                                    ),
+  ('MISC',             'Miscellaneous'                                       ),
+  ('MEGMAG',           'MEG magnetometer'                                    ),
+  ('MEGGRADAXIAL',     'MEG axial gradiometer'                               ),
+  ('MEGGRADPLANAR',    'MEG planar gradiometer'                              ),
+  ('MEGGREFMAG',       'MEG reference magnetometer'                          ),
+  ('MEGREFGRADAXIAL',  'MEG reference axial gradiometer'                     ),
+  ('MEGREFGRADPLANAR', 'MEG reference planar gradiometer'                    ),
+  ('MEGOTHER',         'Any other type of MEG sensor'                        ),
+  ('ECOG',             'Electrode channel: electrocortigogram (intracranial)'),
+  ('SEEG',             'Electrode channel: stereo-electroencephalogram (intracranial)'),
+  ('DBS',              'Electrode channel: deep brain stimulation (intracranial)'),
+  ('AUDIO',            'Audio signal'                                        ),
+  ('PD',               'Photodiode'                                          ),
+  ('EYEGAZE',          'Eye Tracker gaze'                                    ),
+  ('PUPIL',            'Eye Tracker pupil diameter'                          ),
+  ('SYSCLOCK',         'System time showing elapsed time since trial started'),
+  ('ADC',              'Analog to Digital input'                             ),
+  ('DAC',              'Digital to Analog input'                             ),
+  ('HLU',              'Measure position of head and head coils'             ),
+  ('FITERR',           'Fit error signal from each head localization coil'   ),
+  ('OTHER',            'Any other type of channel'                           );
+
+
+-- Insert into physiological_modality
+INSERT INTO physiological_modality
+  (PhysiologicalModality)
+  VALUES
+  ('eeg'),
+  ('meg'),
+  ('ieeg');
+
+
+-- Insert into physiological_status_type
+INSERT INTO physiological_status_type
+  (ChannelStatus)
+  VALUES
+  ('good'),
+  ('bad');
```

Comment:
null?

Reply:
This is a lookup table so there is no need to NULL values. However, in the physiological_channel table (that uses this lookup table), NULL values are allowed. Hope this answers your question.